CHANGELOG.md

```diff
@@ -2,7 +2,14 @@
 
 ## 0.1 DeepGNN
 
-### 0.1.52 - 2022-07-27
+### [Unreleased]
```

Comment:
Remove since unused?

Reply:
It seems people are confused where to put unreleased changes based on the existing PRs. Easier to keep this section at all times.